lib/runner/SMAPIInvoker.js

```diff
@@ -18,6 +18,15 @@ module.exports = class SMAPIInvoker extends invoker.Invoker {
                 + "3) .ask/config file for the project");
         }
 
+        let stage = testSuite.stage;
+        if (!stage) {
+            throw new FrameworkError("To use SMAPI, stage must be specified in one of the following:\n"
```

Comment:
`To use SMAPI, stage must be set to either "developer" or "live" and specified in one of the following:`